### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by colin-axner**
cc @okwme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
+the sha hash of a download or a GPG signature. 
 
 ### Security Notice
 
 If you would like to report a security critical bug related to the relayer repo, please send an email to [`security@cosmosnetwork.dev`](mailto:security@cosmosnetwork.dev)
 
 ## Code of Conduct
 
-The iqlusion team is dedicated to providing an inclusive and harassment free experience for contributors. Please visit [Code of Conduct](CODE_OF_CONDUCT.md) for more information.
+The cosmos organization is dedicated to providing an inclusive and harassment free experience for contributors. Please visit [Code of Conduct](CODE_OF_CONDUCT.md) for more information.
```
Comment (colin-axner) [inline README.md]: not sure what the preferred wording is here. Maybe `cosmos community`?

File: README.md [hunk 2]
```diff
@@ -29,6 +37,37 @@ If you would like to join a relayer testnet, please [check out the instructions]
...
+- relaying a cross chain transfer transaction, its acknowledgement, and timeouts
+- relaying from state
+- relaying from streaming events 
+- sending an UpgradePlan proposal for an IBC breaking upgrade
+- upgrading clients after a counterparty chain has performed an upgrade for IBC breaking changes
+
+The relayer currently cannot:
+- submit IBC client unfreezing proposals 
+- monitor and submit misbehaviour for clients
+- use other IBC light clients such as Solo Machine
```
Comment (AdityaSripal) [inline README.md]:
```suggestion
- use IBC light clients other than Tendermint such as Solo Machine
```

File: README.md [hunk 3]
```diff
@@ -29,6 +37,37 @@ If you would like to join a relayer testnet, please [check out the instructions]
...
+- creating IBC connections
+- creating IBC transfer channels.
+- initiating a cross chain transfer
+- relaying a cross chain transfer transaction, its acknowledgement, and timeouts
+- relaying from state
+- relaying from streaming events 
+- sending an UpgradePlan proposal for an IBC breaking upgrade
+- upgrading clients after a counterparty chain has performed an upgrade for IBC breaking changes
+
+The relayer currently cannot:
```
Comment (AdityaSripal) [inline README.md:53]: Also I believe there is also some inflexibility in creating `ClientState`, eg can't change the `UpgradePath` from its default setting. These should also be listed.

File: README.md (hunk 1 above)
Comment (cwgoes) [reply README.md]: Cosmos should be capitalised. I think "Cosmos community" is fine.

**Review by cwgoes — APPROVED** [review]
ACK modulo minor nit

File: README.md [hunk 4]
```diff
@@ -29,6 +37,38 @@ If you would like to join a relayer testnet, please [check out the instructions]
...
+- create clients with user chosen parameters (such as UpgradePath)
+- submit IBC client unfreezing proposals 
+- monitor and submit misbehaviour for clients
+- use IBC light clients other than Tendermint such as Solo Machine
+- connect to chains which don't implement/enable IBC
+- connect to chains using a different IBC implementation (chain is not using SDK's `x/ibc` module)
```
Comment (fedekunze) [inline README.md]:
```suggestion
- connect to chains using a different IBC implementation (chains not using SDK's `x/ibc` module)
```

**Review by fedekunze — APPROVED** [review]

**Pull request merged**